src/GovLst.sol

```diff
@@ -349,6 +356,8 @@ abstract contract GovLst is IERC20, IERC20Metadata, IERC20Permit, Ownable, Multi
   }
 
   /// @notice The decimal precision which the LST tokens stores its balances with.
+  /// @dev Make sure this matches the Stake token's decimals as an incompatibility
+  /// will create an unintuitive UX where 1 LST token does not equal 1 stake token.
```

Comment:
Something we can do, when we write generalized deploy scripts for this repo, is include a deploy time check that fails if the stake token does match the decimals of the LST tokens. Why don't we create an issue to remind us of that.

Reply:
Created the issue here: https://github.com/withtally/stGOV/issues/80